x-pack/plugin/security/src/test/java/org/elasticsearch/test/SecurityIntegTestCase.java

```diff
@@ -439,7 +440,11 @@ protected void createSecurityIndexWithWaitForActiveShards() {
         );
         CreateIndexRequest createIndexRequest = new CreateIndexRequest(SECURITY_MAIN_ALIAS).waitForActiveShards(ActiveShardCount.ALL)
             .masterNodeTimeout(TEST_REQUEST_TIMEOUT);
-        client.admin().indices().create(createIndexRequest).actionGet();
+        try {
+            client.admin().indices().create(createIndexRequest).actionGet();
+        } catch (ResourceAlreadyExistsException e) {
```

Comment:
Keeping the `createSecurityIndexWithWaitForActiveShards` method here since it's needed, just ignoring any potential exception that may occur if `.security` index already got created.

Reply:
If it already exists, should this also do a get if it's no fully initilized?

Reply:
No, it fails hard during index creation on any exception. 
The reason I kept it, is because only the first test can potentially get into a race condition with a automatic index creation. The sub-sequent ones will not because the index will be deleted after the test and the `QueryableBuiltInRolesSynchronizer` will not attempt to re-create it after it got deleted.

Not attempting to re-create security index when it gets deleted manually is intentional.